test/hotspot/jtreg/runtime/cds/appcds/sharedStrings/SharedStringsHumongous.java

```diff
@@ -27,6 +27,7 @@
  * @summary Use a shared string allocated in a humongous G1 region.
  * @comment -- the following implies that G1 is used (by command-line or by default)
  * @requires vm.cds.write.archived.java.heap
+ * @requires vm.flagless
```

Comment:
Shouldn't 
 @requires vm.gc.G1
works better here if only G1 is supported?

Reply:
This is a direct upstream from Leyden which uses `@requires vm.flagless` although I think you're right.